### ISSUE TITLE
install issue

### ISSUE THREAD

**Issue by r3gisc** [atanas-angelov-dev/vue-router-multiguard#5]
Hi, great lib idea ! but i've tried to install the package and ...

my NPM log: 
```
0/bin/npm" "install" "vue-router-multiguard" "--save"
21 verbose node v8.9.0
22 verbose npm  v6.4.1
23 error Cannot read property 'match' of undefined
24 verbose exit [ 1, true ]
```
my package.json:
```
"dependencies": {
    "@feathersjs/authentication-client": "^1.0.6",
    "@feathersjs/client": "^3.7.3",
    "@feathersjs/socketio-client": "^1.1.4",
    "axios": "^0.18.0",
    "feathers-vuex": "^1.6.2",
    "firebase": "^4.12.1",
    "firebase-tools": "^3.18.4",
    "jsplumb": "^2.8.2",
    "mottle": "^0.8.0",
    "socket.io-client": "^2.1.1",
    "uws": "^99.0.0",
    "vue-feathers": "^0.1.0",
    "vue-i18n": "^7.3.3",
    "vue-images-loaded": "^1.1.2",
    "vue-socket.io-extended": "^3.1.0",
    "vue-vimeo-player": "0.0.6",
    "vuelidate": "^0.6.2",
    "vuex-router": "^1.1.5"
  },
  "devDependencies": {
    "babel-eslint": "^8.2.1",
    "eslint": "^4.18.2",
    "eslint-config-standard": "^11.0.0",
    "eslint-friendly-formatter": "^4.0.1",
    "eslint-loader": "^2.0.0",
    "eslint-plugin-import": "^2.9.0",
    "eslint-plugin-node": "^6.0.1",
    "eslint-plugin-promise": "^3.7.0",
    "eslint-plugin-standard": "^3.0.1",
    "eslint-plugin-vue": "^4.3.0",
    "quasar-cli": "^0.17.0",
    "strip-ansi": "=3.0.1",
    "webpack-auto-inject-version": "^1.1.0"
  },
  "engines": {
    "node": ">= 8.9.0",
    "npm": ">= 5.6.0",
    "yarn": ">= 1.6.0"
  },
  "browserslist": [
    "> 1%",
    "last 2 versions",
    "not ie <= 10"
  ]
```
Thanks in advance :-)

**Comment by atanas-dev** [comment]
Hi @r3gisc ,

This looks like an issue with npm - check out this thread: https://github.com/npm/npm/issues/20954
TL;DR: Delete lockfile and node_modules and run install again.

You can also try installing your modules with Yarn which in my experience is more stable.

**Comment by r3gisc** [comment]
Hi !
I feel stupid ! Of course removing lockfile helped me. It's ok now.
Sorry to have disturbed you. I probably also wanted to thank you for the lib :-)

**Comment by atanas-dev** [comment]
Happy to help :)
